src/lib/commandBuilder.test.ts

```diff
@@ -0,0 +1,100 @@
+import commander from "commander";
+import {
+  build,
+  ICommandBuildElements,
+  validateForRequiredValues
+} from "./commandBuilder";
+
+interface ICommandOption {
+  flags: string;
+  description: string;
+}
+
+describe("Tests Command Builder's build function", () => {
+  it("Sanity tests", () => {
+    const descriptor: ICommandBuildElements = {
+      alias: "cbt",
+      command: "command-build-test",
+      description: "description of command",
+      options: {
+        optionA: {
+          arg: "-a, --optionA <optionA>",
+          description: "description for optionA",
+          required: false
+        },
+        optionB: {
+          arg: "-b, --optionB <optionB>",
+          description: "description for optionB",
+          required: false
+        },
+        optionC: {
+          arg: "-c, --optionC <optionC>",
+          description: "description for optionC",
+          required: false
+        }
+      }
+    };
+
+    const options: ICommandOption[] = Object.getOwnPropertyNames(
```

Comment:
should use `.keys` instead of `.getOwnPropertyNames`

Reply:
refactor the code and this is not needed now. thanks